axlearn/common/param_converter_test.py

```diff
@@ -176,6 +188,26 @@ def test_layer_norm(self):
             ref_layer=hf_layer,
             test_inputs=[inputs],
             ref_inputs=as_torch_tensor(inputs),
+            test_torch_to_axlearn=True,
+        )
+        self.assertNestedAllClose(out, hf_out)
+
+    def test_layer_norm_stateless(self):
```

Comment:
Consider parameterizing the `test_layer_norm` test? It may also make it easier to check the invalid elementwise_affine codepath.

Reply:
Good point. Done.